### PR TITLE
Extend apparmor permissions for /usr/share/openqa/lib/** 

### PR DESCRIPTION
**Pull request by mudler**
See related Progress issue: https://progress.opensuse.org/issues/27934

### CODE REVIEW

**Comment by coolo** [comment]
just glob all of /usr/share/openqa/lib

File: profiles/apparmor.d/usr.share.openqa.script.worker [hunk 1]
```diff
@@ -100,7 +100,7 @@
   /usr/sbin/ipmiconsole rix,
   /usr/share/openqa/lib/DBIx/Class/Timestamps.pm r,
   /usr/share/openqa/lib/OpenQA/** r,
-  /usr/share/openqa/lib/db_helpers.pm r,
+  /usr/share/openqa/lib/** r,
```
Comment (coolo) [inline profiles/apparmor.d/usr.share.openqa.script.worker]: I think this covers also the 2 lines above, but let's leave it as is

**Comment by SergioAtSUSE** [comment]
LGTM

It fixes [poo#27934](https://progress.opensuse.org/issues/27934)

File: profiles/apparmor.d/usr.share.openqa.script.worker (hunk 1 above)
Comment (mudler) [reply profiles/apparmor.d/usr.share.openqa.script.worker]: Yup.. just in case 👽

**Pull request merged**